codecov.yml

```diff
@@ -0,0 +1,9 @@
+# Configuration for codecov.io
+
+comment:
+    layout: "header, changes, diff"
+
+coverage:
+    ignore:
+        - numba/cuda/.*
```

Comment:
it seems to not be ignoring this? https://codecov.io/gh/numba/numba/list/96c18dd50a4f155f9fe23df57f1f026c0e8e696d then sort by coverage